src/dataaggregator/aggregator/policyPoller.go

```diff
@@ -0,0 +1,77 @@
+package aggregator
+
+import (
+	"code.cloudfoundry.org/lager"
+	"dataaggregator/db"
+	"dataaggregator/policy"
+	"time"
+)
+
+type PolicyPoller struct {
+	logger     lager.Logger
+	interval   int
+	database   db.DB
+	tick       *time.Ticker
+	doneChan   chan bool
+	PolicyMap  map[string]*policy.PolicyJson
+	TriggerMap map[string]*policy.Trigger
+}
+
+func NewPolicyPoller(logger lager.Logger, interval int, database db.DB) *PolicyPoller {
+	return &PolicyPoller{
+		logger:     logger,
```

Comment:
I would have created a new logger.WithSession("policy-poller") so we know what this is logging.